src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(../../images/star.svg) no-repeat center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--1 div:nth-child(-n+1),
```

Comment:
Here, just as recommendation, we can use a sass loop [link](https://sass-lang.com/documentation/at-rules/control/for)